app/assets/stylesheets/common/components/buttons.scss

```diff
@@ -322,3 +322,19 @@
     }
   }
 }
+
+// Button with cleared default styles
+// --------------------------------------------------
+.btn-unstyled {
```

Comment:
I'm kinda mixed on this. What's your opinion @jordanvidrine ?

Reply:
I think using `btn-flat` would probably get the styling you want, without needing to make a new style that strips away things as it currently is.

If that doesnt work, this current class and styling is probably fine.

Reply:
Thank you, Jordan! `btn-flat` does the job, also with this style the button looks better in focused state. In Chrome, `btn-unstyled`:

<img width="250" alt="Screenshot 2022-06-16 at 18 00 14" src="https://user-images.githubusercontent.com/1274517/174089201-c4f94826-82e9-4792-8659-6db755f7ed24.png">

`btn-flat`:

<img width="250" alt="Screenshot 2022-06-16 at 17 59 38" src="https://user-images.githubusercontent.com/1274517/174089210-3033c2e9-cf7e-42b6-99de-e852ba617340.png">